tests/MyGame/Example/Color.java

```diff
@@ -7,16 +7,16 @@
  */
 public final class Color {
   private Color() { }
-  public static final byte Red = 1;
+  public static final int Red = 1;
```

Comment:
shouldn't this be `short` ?

Reply:
https://github.com/google/flatbuffers/blob/master/src/idl_gen_java.cpp#L174

Reply:
Fair enough :)